tinyiiod.c

```diff
@@ -164,6 +174,40 @@ void tinyiiod_do_close(struct tinyiiod *iiod, const char *device)
 	tinyiiod_write_value(iiod, ret);
 }
 
+int32_t tinyiiod_do_close_instance(struct tinyiiod *iiod)
```

Comment:
how does `close_instance` differ from `close()` ?

i saw the comment: `tinyiiod_do_close_instance() - Multiple clients can connect simultaneously at tinyiiod, trough libtinyiiod.`  but there is no `open_instance()` or similar behavior anywhere

there isn't any instance ID to identify which instance to close either

Reply:
close() and open() are used for closing and opening a device(like cf-ad9361-dds-core-lpc or ad9361-phy). tinyiiod_do_close_instance() to close a network client, with an "EXIT" command received from iio-oscilloscope(libiio). It doesn't have a pair "OPEN" command that comes form iio-osc, so no pair function. Instance is created at a lower level in network.c, network_accept().